src/test/java/uk/gov/companieshouse/api/accounts/controller/LoansControllerTest.java

```diff
@@ -120,6 +134,21 @@ void createLoanAndServiceThrowsDataException() throws DataException {
         verify(apiResponseMapper, times(1))
                 .getErrorResponse();
     }
+    
+    @Test
+    @DisplayName("Tests the creation of a Loan where the controller returns a bad request binding error for invalid length")
+    void createLoanAndBadRequestResponseServiceThrowsDataException() throws DataException {
```

Comment:
Want to include a similar test for `update`?

Reply:
Ah, good point. Will add one 👍